rialto_airflow/dags/harvest.py

```diff
@@ -164,25 +161,23 @@ def publish(pubs_to_contribs, merge_publications):
 
     sul_pub_jsonl = sul_pub_harvest(snapshot)
 
-    dimensions_dois = dimensions_harvest_dois(snapshot)
+    dimensions_jsonl = dimensions_harvest(snapshot)
 
     openalex_jsonl = openalex_harvest(snapshot)
 
     wos_jsonl = wos_harvest(snapshot)
 
     # TODO: add dimensions_jsonl as a dependency when task is added to DAG
-    openalex_additions = fill_in_openalex(snapshot, openalex_jsonl, wos_jsonl)
+    openalex_additions = fill_in_openalex(snapshot, openalex_jsonl, wos_jsonl)  # noqa: F841
 
     doi_sunet = create_doi_sunet(
-        dimensions_dois,
+        dimensions_jsonl,
         openalex_jsonl,
         sul_pub_jsonl,
         snapshot,
     )
 
-    dimensions_pubs = dimensions_harvest_pubs(doi_sunet, snapshot)
-
-    pubs = merge_publications(sul_pub, openalex_jsonl, dimensions_pubs, snapshot)
+    pubs = merge_publications(sul_pub_jsonl, openalex_jsonl, dimensions_jsonl, snapshot)
```

Comment:
`sul_pub` -> `sul_pub_jsonl` seemed like a small oversight from a prior PR, wasn't inherently part of addressing #162